### PR TITLE
Add map to homepage

### PR DESCRIPTION
**Pull request by jpascual1994**
This PR includes:
- Google map in homepage
- Layout for UsersController

Trello card: https://trello.com/c/XULIfh86/3-3-as-a-user-i-should-be-able-to-see-a-map-so-i-can-easily-identify-targets-in-the-future

### CODE REVIEW

File: app/assets/javascripts/users.js [hunk 1]
```diff
@@ -1,3 +1,11 @@
 // Place all the behaviors and hooks related to the matching controller here.
 // All this logic will automatically be available in application.js.
 // You can use CoffeeScript in this file: http://coffeescript.org/
+
+$(document).ready(function() {
+  var map;
+  map = new google.maps.Map(document.getElementById('map'), {
```
Comment (mdornel) [inline app/assets/javascripts/users.js]: Here you can assign the `map` variable in a single step:
`var map = new google.maps.Map(document.getElementById('map'), {`

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -1,4 +1,6 @@
 class UsersController < ApplicationController
+  layout 'users'
```
Comment (mdornel) [inline app/controllers/users_controller.rb]: Here you don't need to specify the layout. By convention Rails looks for first the specific layout for this controller, if it's not present then uses the application layout. Please check: http://guides.rubyonrails.org/layouts_and_rendering.html Section `2.2.13 Finding Layouts`

File: app/views/layouts/users.html.haml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+!!!
+%html
+  %head
+    %meta{content: 'text/html; charset=UTF-8', 'http-equiv': 'Content-Type'}
+    %meta{ content: 'width=device-width, initial-scale=1', name: 'viewport'}
+    %meta{ content: 'initial-scale = 1.0,maximum-scale = 1.0', name: 'viewport'}
+    %title Target MVD
```
Comment (mdornel) [inline app/views/layouts/users.html.haml]: Add your page title to you I18n dictionary.

File: config/initializers/assets.rb [hunk 4]
```diff
@@ -3,6 +3,7 @@
 # Version of your assets, change this if you want to expire all your assets.
 Rails.application.config.assets.version = '1.0'
 Rails.application.config.assets.precompile += %w( home.css )
+Rails.application.config.assets.precompile += %w( users.css )
```
Comment (mdornel) [inline config/initializers/assets.rb]: It's possible to do this in a single line:
`Rails.application.config.assets.precompile += %w(home.css  users.css)`

File: app/assets/javascripts/users.js [hunk 5]
```diff
@@ -1,3 +1,11 @@
...
+    center: {lat: -34.9011, lng: -56.1645},
```
Comment (mdornel) [inline app/assets/javascripts/users.js]: Maybe you can declare a constant with these values.

**Review by mdornel — CHANGES_REQUESTED** [review]

**Review by mdornel — APPROVED** [review]

File: config/initializers/assets.rb [hunk 6]
```diff
@@ -2,7 +2,7 @@
 
 # Version of your assets, change this if you want to expire all your assets.
 Rails.application.config.assets.version = '1.0'
-Rails.application.config.assets.precompile += %w( home.css )
+Rails.application.config.assets.precompile += %w( home.css users.css )
```
Comment (gabrielrp) [inline config/initializers/assets.rb]: Remove spaces

File: app/views/layouts/users.html.haml [hunk 7]
```diff
@@ -0,0 +1,16 @@
+!!!
+%html
+  %head
+    %meta{content: 'text/html; charset=UTF-8', 'http-equiv': 'Content-Type'}
+    %meta{ content: 'width=device-width, initial-scale=1', name: 'viewport'}
```
Comment (gabrielrp) [inline app/views/layouts/users.html.haml]: Remove space

File: app/views/layouts/users.html.haml [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+    %meta{ content: 'initial-scale = 1.0,maximum-scale = 1.0', name: 'viewport'}
```
Comment (gabrielrp) [inline app/views/layouts/users.html.haml]: Same^

File: app/views/users/homepage.html.haml [hunk 9]
```diff
@@ -13,4 +13,4 @@
   .img-footer
     = image_tag 'smilies.png', class: 'smiles-img'
 .home-right
-  MAPA
+  .mymap#map
```
Comment (gabrielrp) [inline app/views/users/homepage.html.haml]: Prefer to put the id first: `#map.mymap`

File: app/views/layouts/users.html.haml [hunk 10]
```diff
@@ -0,0 +1,16 @@
...
+    %title
+      = I18n.t(:target_title)
+    = stylesheet_link_tag 'https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css'
+    = stylesheet_link_tag   'application', media: 'all'
```
Comment (gabrielrp) [inline app/views/layouts/users.html.haml]: Same^

**Review by gabrielrp — CHANGES_REQUESTED** [review]

**Review by gabrielrp — APPROVED** [review]

**Pull request merged**